Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39974&attempt_id=1f75d7d8-b228-415b-8d60-038949ff3250)

Attempted: simplescreenrecorder

<details><summary>Partial log (click to expand)</summary><p>

```
../glinject/elfhacks.h:67:3: error: #error neither __elf32 nor __elf64 is defined
 # error neither __elf32 nor __elf64 is defined
   ^~~~~
[3/55] Building CXX object glinject/CMakeFiles/ssr-glinject.dir/GLInject.cpp.o
[4/55] Building CXX object glinject/CMakeFiles/ssr-glinject.dir/GLXFrameGrabber.cpp.o
[5/55] Building CXX object glinject/CMakeFiles/ssr-glinject.dir/SSRVideoStreamWriter.cpp.o
[6/55] Automatic MOC for target simplescreenrecorder
ninja: build stopped: subcommand failed.
builder for '/nix/store/92ipkhyaa5wyjqxpjdfkxykh89xs01ak-simplescreenrecorder-0.3.10.drv' failed with exit code 1
[31;1merror:[0m build of '/nix/store/92ipkhyaa5wyjqxpjdfkxykh89xs01ak-simplescreenrecorder-0.3.10.drv' failed
```
</p></details>

--- a/pkgs/applications/video/simplescreenrecorder/default.nix
+++ b/pkgs/applications/video/simplescreenrecorder/default.nix
@@ -1,5 +1,5 @@
-{ stdenv, fetchurl, alsaLib, ffmpeg, libjack2, libX11, libXext
-, libXfixes, libGLU_combined, pkgconfig, libpulseaudio, qt4, cmake, ninja
+{ stdenv, fetchurl, alsaLib, ffmpeg, libjack2, libX11, libXext, qtx11extras
+, libXfixes, libGLU_combined, pkgconfig, libpulseaudio, qtbase, cmake, ninja
 }:
 
 stdenv.mkDerivation rec {
@@ -10,6 +10,8 @@ stdenv.mkDerivation rec {
     url = "https://github.com/MaartenBaert/ssr/archive/${version}.tar.gz";
     sha256 = "02rl9yyx3hlz9fqvgzv7ipmvx2qahj7ws5wx2m7zs3lssq3qag3g";
   };
+
+  cmakeFlags = [ "-DWITH_QT5=TRUE" ];
 
   patches = [ ./fix-paths.patch ];
 
@@ -24,14 +26,14 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig cmake ninja ];
   buildInputs = [
     alsaLib ffmpeg libjack2 libX11 libXext libXfixes libGLU_combined
-    libpulseaudio qt4
+    libpulseaudio qtbase qtx11extras
   ];
 
   meta = with stdenv.lib; {
     description = "A screen recorder for Linux";
     homepage = http://www.maartenbaert.be/simplescreenrecorder;
     license = licenses.gpl3;
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
     maintainers = [ maintainers.goibhniu ];
   };
 }

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -4878,7 +4878,7 @@ with pkgs;
 
   simpleproxy = callPackage ../tools/networking/simpleproxy { };
 
-  simplescreenrecorder = callPackage ../applications/video/simplescreenrecorder { };
+  simplescreenrecorder = libsForQt5.callPackage ../applications/video/simplescreenrecorder { };
 
   sipsak = callPackage ../tools/networking/sipsak { };
 

